onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -25,7 +25,6 @@
 from transformers import LlamaConfig, LlamaForCausalLM, LlamaTokenizer
 
 import onnxruntime as ort
-from onnxruntime.transformers.benchmark_helper import measure_memory
```

Comment:
I feel this is desired way to import benchmark_helper use full path. We could use this benchmark script from any working directory we want.

Reply:
Changed back to full path